GrizzFit/src/screens/LoginScreen.js

```diff
@@ -0,0 +1,112 @@
+import { useNavigation } from '@react-navigation/native'
+import React, { useContext, useState } from 'react'
+import { KeyboardAvoidingView, StyleSheet, Text, View } from 'react-native'
+import { TextInput, TouchableOpacity } from 'react-native-gesture-handler'
+import { AuthContext } from '../navigation/AuthProvider'
+
+import { darkModePalette } from '../styles/DarkModePalette'
+
+const LoginScreen = () => {
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+
+    const {login} = useContext(AuthContext);
+
+    const navigation = useNavigation()
+
+    const navRegister = () => {
+        navigation.navigate("Register")
+    }
+
+    return (
+        <KeyboardAvoidingView
+            style={styles.container}
+            behavior="padding"
+        >
+            <View style={styles.inputContainer}>
+                <Text style={styles.title}>LOGIN</Text>
+                <TextInput
+                    placeholder="Email"
+                    value={email}
+                    onChangeText={text => setEmail(text)}
+                    style={styles.input}
+                />
+                <TextInput
+                    placeholder="Password"
+                    value={password}
+                    onChangeText={text => setPassword(text)}
+                    style={styles.input}
+                    secureTextEntry
```

Comment:
So I was looking at these TextInputs and saw that there's a `textContentType` property you can set which changes the keyboard that appears and affects how the OS can autofill in these fields. 

You could do use the following for the different TextInputs you're using across these components:
```js
textContentType="name"
textContentType="emailAddress"
textContentType="password"
```

Reply:
Here's a list of all the possible content types for the TextInput:
```js
'none'
'URL'
'addressCity'
'addressCityAndState'
'addressState'
'countryName'
'creditCardNumber'
'emailAddress'
'familyName'
'fullStreetAddress'
'givenName'
'jobTitle'
'location'
'middleName'
'name'
'namePrefix'
'nameSuffix'
'nickname'
'organizationName'
'postalCode'
'streetAddressLine1'
'streetAddressLine2'
'sublocality'
'telephoneNumber'
'username'
'password'
'newPassword'
'oneTimeCode'
```